samples_project/Assets/SampleViewer/Samples/Measure/Measure.cs

```diff
@@ -26,6 +26,7 @@ public class Measure : MonoBehaviour
     public float InterpolationInterval = 100;
```

Comment:
The interpolation interval being constant results in different behavior for the interpolation points depending on what unit is being used. Does not show any interpolation points for miles/km.

![Unity_ve6gT1ggdT](https://github.com/Esri/arcgis-maps-sdk-unity-samples/assets/16495151/095d1b7c-3138-4ac3-ab05-8c1e5d2dd41b)